magclassic/site_sections/treasury.py

```diff
@@ -0,0 +1,19 @@
+from magclassic import *
```

Comment:
Since this is now at a different URL, we'll need to change the configuration in the Dept Checklist after this is merged.  I'll take care of that over the weekend, since I need to update the config to change due dates anyway.